scripts/hail_batch/hgdp1kg_tobwgs_pca/README.md

```diff
@@ -0,0 +1,9 @@
+# Perform a pca on the combined HGDP + 1KG and TOB-WGS data
+
+This runs a Hail query script in Dataproc using Hail Batch in order to perform a pca on the 1KG + HGDP + TOB-WGS dataset. To run, use conda to install the analysis-runner, then execute the following command:
+
+```sh
+analysis-runner --dataset ancestry \
+--access-level test --output-dir "gs://cpg-ancestry-temporary/1kg_hgdp_tobwgs_pca/v0" \
```

Comment:
Should we write this to the `analysis` bucket?

Reply:
so instead have: `--output-dir "gs://cpg-ancestry-analysis/1kg_hgdp_tobwgs_pca/v0"` ?

Reply:
Yes, especially given that any results in `temporary` will disappear. The result of this analysis looks like something we might want to revisit later?

Reply:
I've now changed this to the analysis bucket